src/rt/atomic.rs

```diff
@@ -535,7 +531,7 @@ impl State {
         if let Some(loaded) = current.ahead(&self.unsync_loaded_at) {
             location::panic("Causality violation: Concurrent `unsync_load` and mut accesses.")
                 .location("created", self.created_location)
-                .thread("mut", loaded, self.unsync_loaded_locations[loaded])
+                .thread("with_mut", loaded, self.unsync_mut_locations[loaded])
```

Comment:
I think this tests concurrent `unsync_load`, so the two conflicts should be "unsync load 1" and "unsync load 2"?

Reply:
Hmm, why would two `unsync_load`s race? I wonder if it's the `ahead` check that's wrong, and should be checking `unsync_mut_at`?